packages/graphql/lib/src/link/link.dart

```diff
@@ -26,7 +27,7 @@ Link _concat(
   });
 }
 
-class Link {
+class Link extends Equatable {
```

Comment:
>  hint • This class (or a class which this class inherits from) is marked as '@immutable', but one or more of its instance fields are not final: Link.request at lib/src/link/link.dart:30:7 • must_be_immutable

Reply:
Let me work on that, we still need a way of testing that `link.from([link1, link2, link3])` is equal to `link1..concat(link2)..concat(link3)`. Maybe override equal behavior as Equatable does.